src/components/localization/localized-link.js

```diff
@@ -141,6 +141,8 @@ const affiliate_links = ['affiliate_sign_in', 'affiliate_sign_up']
 const deriv_app_links = ['dbot', 'deriv_app', 'mt5', 'derivx']
 const deriv_other_products = ['binary', 'smart_trader']
 const deriv_social_platforms = ['blog', 'community', 'developers', 'zoho']
+// add item to this array if you need to make external link without modal window
+const deriv_new_window_links = ['terms-and-conditions']
```

Comment:
We don't need to add a new type for this as this will just be treated like another page of our site. Hence, no changes are required to our `localized-link.js`.